ts/Series/Lollipop/LollipopPointOptions.d.ts

```diff
@@ -14,15 +14,22 @@
  *
  * */
 
-import type DumbbellPointOptions from '../Dumbbell/DumbbellPointOptions';
+import type ScatterPointOptions from '../Scatter/ScatterPointOptions';
+import type ColorType from '../../Core/Color/ColorType';
+import DashStyleValue from '../../Core/Renderer/DashStyleValue';
 
 /* *
  *
  *  Declarations
  *
  * */
 
-export interface LollipopPointOptions extends DumbbellPointOptions {
+export interface LollipopPointOptions extends ScatterPointOptions {
+    connectorColor?: ColorType;
+    connectorWidth?: number;
+    dashStyle?: DashStyleValue;
+    low?: number;
+    lowColor?: ColorType;
```

Comment:
Why `low`? Is this a rudiment from the dumbbell inheritance chain? In that case, I think we should change it. Looking at the `highcharts/demo/lollipop/demo.js`, it makes no sense to call the value `low`, when the chart in all other aspects works like a standard column chart.

Reply:
Yes, this is a rudiment from the dumbbell inheritance. Sure, it would be more clear if we remove this property and leave it only with the possibility to use the y property - just like in the column charts.

Reply:
But should we deprecate it (same as `lowColor`) and leave function `toYData` as it is now to keep it working with `low` property?